WaterSkiBaan/Program.cs

```diff
@@ -139,20 +148,20 @@ static public void SportersGaanNaarInstructie(object sender, SportersEventArgs a
             }
         }
 
-        static void VoegRandomSportersToe(Queue<Sporter> queue, int aantal)
-        {
-            for (int i = 0; i < aantal; i++)
-            {
-                if ((i % 2) == 0)
-                {
-                    queue.Enqueue(new Wakeboarder());
-                }
-                else
-                {
-                    queue.Enqueue(new Skier());
-                }
-            }
-        }
+        //static void VoegRandomSportersToe(Queue<Sporter> queue, int aantal)
+        //{
+        //    for (int i = 0; i < aantal; i++)
+        //    {
+        //        if ((i % 2) == 0)
+        //        {
+        //            queue.Enqueue(new Wakeboarder());
+        //        }
+        //        else
+        //        {
+        //            queue.Enqueue(new Skier());
+        //        }
+        //    }
+        //}
 
         static void VoegRandomUitrustingToe(IOpslag stapel, int aantal, SportUitrustingType type)
```

Comment:
"Comment de methodes die de opslagen en de wachtrijen vullen."

Moet deze methode dan ook niet gecomment worden? Alhoewel er niet letterlijk 'VulOpslag' in de naam van de methode staat zoals de methode hieronder is dat wel wat er gebeurd.

Reply:
VoegRandomUitrustingToe bedoel je? Ik denk het niet, die wordt op andere plaatsen namelijk aangeroepen. Uiteindelijk wordt de opslag ook niet gevuld maar wordt er een object afgegeven, zoals skies.

Reply:
Ik ga de PR completen. Mocht je het er toch niet mee eens zijn, let me know!